src/Composer.vala

```diff
@@ -79,12 +79,23 @@ public class Mail.Composer : Hdy.ApplicationWindow {
     }
 
     construct {
+        cc_button = new Gtk.ToggleButton.with_label (_("Cc"));
```

Comment:
Also the cc_button is focused and highlighted by default - at least for me - which might be confusing for users so maybe `set_focus (to_val);` seems to be a reasonable default